charts/admin-panel/values.yaml

```diff
@@ -21,3 +21,12 @@ secrets:
   githubOauthClientSecret: ""
 otelExporterOtlpEndpoint: http://localhost:4318
 tracingServiceName: "admin-panel"
+labels:
+  - kube-monkey/enabled: enabled
+  - kube-monkey/identifier: admin-panel
+  - kube-monkey/kill-mode: fixed
+  - kube-monkey/kill-value: "1"
+  - kube-monkey/mtbf: "3"
+podLabels:
+  - kube-monkey/enabled: enabled
+  - kube-monkey/identifier: admin-panel
```

Comment:
Both of these should be objects, not arrays. You can validate the syntax by using `helm template admin-panel .` in `charts/charts/admin-panel`

Reply:
```
% helm template .
Error: template: admin-panel/templates/secrets.yaml:12:54: executing "admin-panel/templates/secrets.yaml" at <b64enc>: invalid value; expected string
```

Reply:
Pushed a fix, should work now.

Reply:
if I commented the value I could get it to work